### PR TITLE
Igvf 316 add grcm39 enum

### PR DESCRIPTION
**Pull request by corismall**

### CODE REVIEW

File: src/igvfd/schemas/changelogs/gene.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`gene.json`*
 
+### Minor changes since schema version 3
+
+* Add `GRCM39` to `dependentSchemas` and `locations` assemblies
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/gene.md:5]: there is no need to mention "dependent schemas". It is just logic of dependencies, and I don't think should be included in the changelog.

**Review by gabdank — CHANGES_REQUESTED** [review]
minor comment about changelog

File: src/igvfd/schemas/changelogs/gene.md (hunk 1 above)
Comment (corismall) [reply src/igvfd/schemas/changelogs/gene.md:5]: okay i'll remove it

File: src/igvfd/schemas/changelogs/gene.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`gene.json`*
 
+### Minor changes since schema version 3
+
+* Add `GRCM39` to `locations` assemblies
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/gene.md:5]: I propose to change to :   * Add `GRCM39` to `locations.assembly`

**Review by gabdank — CHANGES_REQUESTED** [review]
minor comment on changelog

File: src/igvfd/schemas/changelogs/gene.md (hunk 2 above)
Comment (gabdank) [reply src/igvfd/schemas/changelogs/gene.md:5]: And I also think it is GRCm39 and not GRCM39

File: src/igvfd/schemas/changelogs/variant.md [hunk 3]
```diff
@@ -1 +1,6 @@
 ## Changelog for variant.json
+
+### Minor changes since schema version 1
+
+* Remove `mm10` from assembly
+* Add `GRCM39` to assembly
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/variant.md:6]: GRCm39

**Review by gabdank — CHANGES_REQUESTED** [review]
changelog comments and GRCm39 instead of GRCM39

**Review by gabdank — APPROVED** [review]

**Pull request merged**